File: src/ShortStackLogic/Models/StackPullRequestComment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Microsoft.Tools.Productivity.ShortStack
{
    [DataContract]
    public class StackPullRequestComment
    {
        /// <summary>
        /// Content of this comment
        /// </summary>
        [DataMember(Name = "content")]
        public string Content { get; set; }

        /// <summary>
        /// Author name of this comment
        /// </summary>
        [DataMember(Name = "author")]
        public string Author { get; set; }

        /// <summary>
        /// Date of the comment
        /// </summary>
        [DataMember(Name = "date")]
        public DateTime Date { get; set; }
    }
}

File: src_OLD/VSCodeExtension/src/JsonRpcTypes/StackPullRequestComment.ts
/* --------------------------------------------------------------------------------------------
 * Copyright (c) Microsoft Corporation. All Rights Reserved.
 * ------------------------------------------------------------------------------------------ */

 namespace JsonRpcTypes {
    export interface StackPullRequestComment {
        /**
         * Content of this comment
         */
        content: string;

        /**
         * Author name of this comment
         */
        author: string;

        /***
         * Date of the comment
         */
        date: Date;
    }
}